src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -143,6 +149,16 @@ public boolean doneScoring() {
   public void initialize() {
     armEncoder.setPosition(
         armAbsoluteEncoder.getPosition() + Calibrations.ARM_ABSOLUTE_ENCODER_OFFSET_DEG);
+
+    double elevatorDutyCycleEncodersDifference =
+        elevatorDutyCycleEncoderOne.getAbsolutePosition() * Constants.REVOLUTIONS_TO_DEGREES
+            - elevatorDutyCycleEncoderTwo.getAbsolutePosition() * Constants.REVOLUTIONS_TO_DEGREES;
+    if (elevatorDutyCycleEncodersDifference < 0) {
```

Comment:
0 is an integer, we should compare with 0.0 since `elevatorDutyCycleEncodersDifference ` is a double. This probably generates the same code but is clearer